main.go

```diff
@@ -7,6 +7,11 @@ import (
 	"github.com/labstack/echo/v4"
 	"github.com/labstack/echo/v4/middleware"
 	
```

Comment:
Try to use go fmt before checking in, most IDE have the ability to do this on save. This will sort the imports alphabetically by grouping. it is also a good idea to have External together (Like you have above here) and Internal(below) which is fantastic!